python/tests/test_modify_changelog.py

```diff
@@ -295,3 +295,102 @@ def test_custom_changelog_path():
         # Check changelog was updated
         updated_changelog = custom_changelog_path.read_text()
         assert "- Add new feature [#456](https://github.com/egraphs-good/egglog-python/pull/456)" in updated_changelog
+
+
+def test_update_changelog_with_markdown_backticks():
+    """Test that PR titles with markdown backticks are preserved correctly."""
```

Comment:
Can you consolidate these into one test with pytest params? And also if any tests above can also be consolidated with these easily please do that to reduce duplication.